Userland/Libraries/LibCore/System.h

```diff
@@ -46,6 +46,11 @@ ErrorOr<void> disown(pid_t pid);
 ErrorOr<void> profiling_enable(pid_t, u64 event_mask);
 ErrorOr<void> profiling_disable(pid_t);
 ErrorOr<void> profiling_free_buffer(pid_t);
+enum class SearchInPATH {
```

Comment:
nit: `SearchInPath`

I don't think the style violation is worth it just so the case matches the `PATH` env variable. :smiley:

Reply:
Fair enough! Originally I named the `filename` parameter `path` so it was confusing, I should have changed it back.